doc/classes/String.xml

```diff
@@ -246,14 +270,14 @@
 			<argument index="0" name="to" type="String">
 			</argument>
 			<description>
-				Perform a case-sensitive comparison to another string, return -1 if less, 0 if equal and +1 if greater.
+				Performs a case-sensitive comparison to another string, return -1 if less, 0 if equal and +1 if greater.
```

Comment:
- comma to period
"Returns [code]-1[/code] if less than, [code]+1[/code] if greater than, or [code]0[/code] if equal.